index.html

```diff
@@ -10,29 +10,25 @@
 <body>
     <div class="header"><h2>STUDENTS EXPENSE TRACKER </h2></div>
     <div class="play">
-       <form action="" class="example"> <input type="text" autocomplete="off" name="name" id="text" placeholder=" Name" class="text-input" >
```

Comment:
Never use form tags when you are not submitting data to a server. I have removed it, Doing Form tags without having a server is why you get the refresh issue when you click submit

Reply:
Also you only need one form tag to hold a group of inputs and not multiple form tags (if you need to use them)